src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/OidLookup.cs

```diff
@@ -206,9 +206,9 @@ static void AddEntry(string oid, string primaryFriendlyName, string[]? additiona
             AddEntry("1.3.133.16.840.63.0.2", "ECDH_STD_SHA1_KDF");
             AddEntry("1.3.132.1.11.1", "ECDH_STD_SHA256_KDF");
             AddEntry("1.3.132.1.11.2", "ECDH_STD_SHA384_KDF");
-            AddEntry("1.2.840.10045.3.1.7", "ECDSA_P256", new[] { "nistP256", "secP256r1", "x962P256v1" } );
-            AddEntry("1.3.132.0.34", "ECDSA_P384", new[] { "nistP384", "secP384r1" });
-            AddEntry("1.3.132.0.35", "ECDSA_P521", new[] { "nistP521", "secP521r1" });
+            AddEntry("1.2.840.10045.3.1.7", "ECDSA_P256", new[] { "nistP256", "secP256r1", "x962P256v1", "ECDH_P256" } );
```

Comment:
This is a strictly unrelated change to case-sensitive on Windows, but it's something I identified while trying to write tests. On non-Windows, we currently don't map ECDH_ algorithm identifiers like we do for ECDSA_.

```C#
Console.WriteLine(ECCurve.CreateFromFriendlyName("ECDH_P256").Oid.Value); // prints nothing because it's null
Console.WriteLine(ECCurve.CreateFromFriendlyName("ECDSA_P256").Oid.Value); // prints 1.2.840.10045.3.1.7
```

So this aliases ECDH_P256 => 1.2.840.10045.3.1.7 (and the other NIST ECDH curves) so the friendly name lookup succeeds.